package.json

```diff
@@ -3,8 +3,13 @@
   "version": "2.1.0",
   "description": "k-neareset neighbors search for RBush",
   "main": "index.js",
+  "jsdelivr": "rbush-knn.js",
+  "unpkg": "rbush-knn.js",
   "scripts": {
-    "test": "eslint *.js && tape test.js"
+    "test": "eslint *.js && tape test.js",
```

Comment:
The build fails because ESLint is picking up the built files. Let's make this line more specific.